app/factories/flood_risk_engine/steps/form_object_factory.rb

```diff
@@ -42,6 +42,7 @@ def setup_form_object(step)
           begin
             form_name.constantize
           rescue NameError => x
```

Comment:
while we're on about time lost an d confusign developers, can we please remove that generic.yaml crap .. that's not production data and has lost me hours baffled why locale data was showing up wierd